### PR TITLE
Add digi space register cmd

### PR DESCRIPTION
**Pull request by vakumar1**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
```

### CODE REVIEW

File: cmd/digi/space/command.go [hunk 1]
```diff
@@ -133,6 +134,36 @@ func StopController(name string) error {
 	return helper.RunMake(nil, "stop-"+name, true, false)
 }
 
+var registerCmd = &cobra.Command{
+	Use:		"register REGISTRY USER",
+	Short:		"register the current dSpace on the given registry",
+	Aliases:	[]string{"register"},
+	Args:		cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		kc, err := k8s.LoadKubeConfig();
```
Comment (silveryfu) [inline cmd/digi/space/command.go:143]: Can remove ";"? Same below

File: scripts/register-space.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+source ./.env
+
+# temporary: use cluster-ip address for apiserver
+# TODO: expose apiserver using load-balancer
+APISERVER_IP=$(kubectl -n kube-system get pod -l component=kube-apiserver -o=jsonpath="{.items[0].metadata.annotations.kubeadm\.kubernetes\.io/kube-apiserver\.advertise-address\.endpoint}")
```
Comment (silveryfu) [inline scripts/register-space.sh:6]: I wonder if this IP is available in the kubeconfig? If so, might be easier to pass it in?

**Review by silveryfu — CHANGES_REQUESTED** [review]
Looks good to me overall! Nice work. Left only a comment on APISERVER_IP.

A final nit: could you run `make fmt`and `git status` check if there are formatting changes (if there is, pls add to the PR)?

File: scripts/register-space.sh (hunk 2 above)
Comment (vakumar1) [reply scripts/register-space.sh:6]: It looks like this IP isn't in the kubeconfig, only in the details for the pod running the apiserver.

Comment (silveryfu) [reply scripts/register-space.sh:6]: Thanks for checking @vakumar1  - sorry, by IP I meant a URL. There is a URL in the kubeconfig for the apiserver? Could you check if one can send request to that URL (in place of the APISERVER_IP obtained above)?

**Comment by silveryfu** [comment]
LGTM. Thanks @vakumar1. 

> Use the URL in kubeconfig in place of `API_SERVER` obtained via `kubectl`

It'd be nice to confirm this and if so, we can use kubeconfig to pass in the URL and do the change in a future PR.

**Pull request merged**